duckduckgo/Android
# Format project source

Task/Issue URL: N/A

**Description**:
The kotlin formatter has changed a number of times over the course of this project. This PR brings the project's formatting in line with the latest kotlin formatting style.

**Steps to test this PR**:
1. Run the formatter over the entire project except the cpp/third-party folder
1. Check to see that files stay the same (other than exception mentioned in the notes below)

NOTES: The xml resource licenses get updated in a way that isn't an improvement, so I left these alone. It would be great to update our formatter to NOT make that update in the future.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
